### ISSUE TITLE
Flaky-test: ExtensibleLoadManagerTest.testStopBroker

### ISSUE THREAD

**Issue by lhotari** [apache/pulsar#21555]
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.

### Example failures

- [2023-11-06T06:47:12.1657116Z](https://github.com/apache/pulsar/actions/runs/6766900442/job/18389187067#step:12:26496) 

### Exception stacktrace

```
java.lang.AssertionError: did not expect [pulsar://pulsar-broker-0:6650] but found [pulsar://pulsar-broker-0:6650]
	at org.testng.Assert.fail(Assert.java:110)
	at org.testng.Assert.failEquals(Assert.java:1581)
	at org.testng.Assert.assertNotEqualsImpl(Assert.java:156)
	at org.testng.Assert.assertNotEquals(Assert.java:2197)
	at org.testng.Assert.assertNotEquals(Assert.java:2219)
	at org.apache.pulsar.tests.integration.loadbalance.ExtensibleLoadManagerTest.testStopBroker(ExtensibleLoadManagerTest.java:271)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
	at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
	at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

```

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by nikam14** [comment]
Fixed in #21811

**Comment by lhotari** [comment]
fixed by #21811
